cmd/diff/cmddiff.go

```diff
@@ -19,7 +19,7 @@ import (
 func NewCmdDiff(f util.Factory, ioStreams genericclioptions.IOStreams) *cobra.Command {
 	options := diff.NewDiffOptions(ioStreams)
 	cmd := &cobra.Command{
-		Use:                   "diff DIRECTORY",
+		Use:                   "diff (DIRECTORY | STDIN)",
```

Comment:
I find that this fails in what appears to be `common.ExpandPackageDir` and `common.expandDir`:

> % ... | kpt live diff   
> error: lstat -: no such file or directory

It looks like we need to thread this treatment of the single hyphen (meaning standard input) deeper into the call path.

Reply:
See #170.